src/Native/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -643,6 +628,26 @@ REDHAWK_PALEXPORT bool REDHAWK_PALAPI PalStartBackgroundWork(_In_ BackgroundCall
     int st2 = pthread_attr_destroy(&attrs);
     ASSERT(st2 == 0);
 
+    if (highPriority && st == 0)
+    {
+        int policy = SCHED_RR;
+        int maxPriority = sched_get_priority_max(policy);
+
+        if (maxPriority > 0)
+        {
+            maxPriority = maxPriority * 9 / 10;
```

Comment:
It seems it would be good to make this ratio configurable and maybe the 90% is too high. On Windows, we set the priority to THREAD_PRIORITY_HIGHEST, which for threads that are running in normal priority class gets it about 32% of the maximum possible priority while the normal threads run at about 26% of the maximum priority. Windows has even higher priority called THREAD_PRIORITY_TIMECRITICAL, which for normal priority class gets 50% of the maximum. You can see the overview of the priorities here: https://docs.microsoft.com/en-us/windows/win32/procthread/scheduling-priorities. The way Windows and Linux handle thread priorities is most likely different, but it still makes me think this value will need to be fine-tuned based on some performance benchmarks running with server GC on, like ASP.NET benchmarks.

Reply:
I will add a config variable for it and reduce the default value. Any suggestions for the name?